docs/datasync/datasync-soft-delete.md

```diff
@@ -46,8 +48,19 @@ type Comment {
   createdAt: String
   updatedAt: String
 }
+```
+
+The `@delta` annotation adds a `sync` query or a delta query:
+```graphql
+type Query {
+  syncComments(lastSync: String!, filter: CommentFilter): CommentDeltaList!
+}
+```
 
+This allows you to get all the changed(updates and deletes) documents in a collection since the `lastSync` timestamp. Internally this uses the `updatedAt` database field to check if any documents in the database have been modified, by comparing client provided `lastSync` timestamp value. The documents to be returned in a `sync` query may also be filtered, so you can, for example, only `sync` comments on a specific post. last stuff 
```

Comment:
I was lost when reading this sentence:  `The documents to be returned in a `sync` query may also be filtered, so you can, for example, only `sync` comments on a specific post. last stuff ` 

What did you want to say here?

Reply:
Good point. This can be simplified